tutorial/kalman/tutorial-ukf-nonlinear-example.cpp

```diff
@@ -0,0 +1,339 @@
+/****************************************************************************
+ *
+ * ViSP, open source Visual Servoing Platform software.
+ * Copyright (C) 2005 - 2023 by Inria. All rights reserved.
+ *
```

Comment:
2005 - 2024

Reply:
Done in 7b87be38904131169258f4a897112fed9c0c43e6